.github/workflows/ci.yml

```
@@ -50,3 +53,49 @@ jobs:
+          if [ -n "$offenders" ]; then
+            echo "::error::A data-labeled PR may not edit shared files. Offending paths:"
+            echo "$offenders" | sed 's/^/  /'
+            echo ""
+            echo "Put per-PR labelling on the dataset card (domain_by_pr / task_type_by_pr /"
+            echo "linked_issues_by_pr), add repo-specific tests to tests/test_overrides_<repo>.py,"
+            echo "document the repo in docs/source-repos/<repo>.md, and let STATUS.md regenerate."
+            echo "If this really is a pipeline or schema change, add the 'pipeline' or 'schema' label."
+            exit 1
+          fi
+          echo "No shared files touched."
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)